.circleci/config.yml

```diff
@@ -403,6 +395,11 @@ defaults:
         name: win/default
         shell: cmd.exe
 
```

Comment:
You replaced `base_win_cmd` with the `base_win_bash` in the `b_bytecode_win` and it is not being used anymore. Can we remove it now?

Reply:
Good point. I thought it was still in use by other jobs but apparently not. Removed.